Playbooks/Put-MDEAlert-Hunting-GitHub/readme.md

```diff
@@ -0,0 +1,78 @@
+# Tutorial: Push Defender for Endpoint Alerts including Alert Names, MITRE Tactics, Techniques and Sub-Techniques as Hunting ARM Templates into a Sentinel Github Repository 
+
+Intent: As a threat hunter or SecOps analyst, I want to be able to easily hunt for threats pertaining to certain Mitre tactics or techniques in Sentinel. 
+
+Many of these detections are already covered under Defender for Endpoint, and it would be helpful to easily hunt for certain type of Defender for Endpoint alerts by their associated MITRE tactics, techniques and sub-techniques within the Microsoft Sentinel Hunting interface.
+
+   ![Sentinel](./Media/Sentinel-Hunting.png)
+   ![MDE](./Media/MDE-alert.png)
+
+You can see a sample of a generated JSON ARM Hunting Template for Sentinel here - [MDE Alert JSON ARM Hunting Template](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Playbooks/Put-MDEAlert-Hunting-GitHub/Media/MDE_A_process_was_injected_with_potentially_malicious_code_T1055_T1055_001_T1055_002_T1055_003_T1055_004_T1055_012_T1059_001.json)
+
+## Prerequisites
+
+You will need an existing Microsoft Sentinel workspace, connected to a GitHub repository for [Deploying Custom Content from your repository](https://learn.microsoft.com/en-us/azure/sentinel/ci-cd?tabs=github).
+
+You will need [Defender for Endpoint alerts Syncronized to Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/microsoft-365-defender-sentinel-integration) which populates the required SecurityAlerts table in Sentinel. 
+
+You will need to generate a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for your Github Repository associated with Sentinel with the Repo, and Workflow permissions.
+
+You will need your Personal Access Token from GitHub stored in Keyvault and have the URL of the secret handy.
```

Comment:
@SCStelz , By URL of secret, do you mean by the URL present inside the secret like mentioned in screen shot below .
![image](https://user-images.githubusercontent.com/97503740/211983484-1a448e14-7716-4e0b-b303-309215d6182a.png)

if Yes , then please mention the steps as well on how to fetch the url of the secret. so that user who will try to use your playbook will not get confused.

If no, please justify